CONTRIBUTING.md

```diff
@@ -17,6 +17,8 @@ When you're making your contribution(s) to Glosario, please:
 
 To add a new entry, please [fork][forking-guide] the [main Glosario repository][repo] and, on a new branch, add the term and definition to [`glossary.yml`][glossary]. This glossary file is written in [YAML].
 
+When adding a new term or translation, please notice the indentation on the YAML file.
```

Comment:
```suggestion
When adding a new term or translation, please take care with the indentation on the YAML file. Indentation is syntactically significant in YAML.
```

Perhaps something like this? I wonder if it would be beneficial to point contributors towards a simple online YAML validator like http://www.yamllint.com/ too?

Reply:
Hi @tobyhodges , I think your suggestion  of change is good! About the validator, I think we could add but maybe would need to say that people should use to validate, but don't copy the generated yaml.

Ex: I tried validating a part of the code from #52 (just to test the validator)

the original code:
```
  pt:
    term: "caminho relativo"
    def: >
      Um caminho cujo destino é interpretado de maneira relativa a outro
      local, como o [diretório de trabalho](#current_working_directory). Um caminho
      relativo é o equivalente a indicar um destino com termos como "siga em frente" 
      e "vire a esquerda".
  ref: 
    - absolute_path
```

After I clicked "Go", apeard a message of **Valid YAML!**, but the YAML that I pasted in the validator was changed to this code (with different identation that we need): 

```
--- 
pt: 
  def: "Um caminho cujo destino é interpretado de maneira relativa a outro local, como o [diretório de trabalho](#current_working_directory). Um caminho relativo é o equivalente a indicar um destino com termos como \"siga em frente\"  e \"vire a esquerda\".\n"
  term: "caminho relativo"
ref: 
  - absolute_path
```

So if we add a validator, this should be also something to beware... what do you think?

Reply:
Or maybe we can find a validator that does not change the YAML code

Reply:
Yes, if we can't suggest a validator that will preserve our chosen level of indentation I think it's better not to mention one at all.

Reply:
Yes.. If changes the identation and people copy and paste it from the validator, it will create more errors

Reply:
https://codebeautify.org/yaml-validator found this option in google that does not seem to change the code

Reply:
If it helps, the tests on this repository check if the yaml is valid, so if they submit invalid yaml, an error will be reported with information about how it can be fixed.

Reply:
Thanks @zkamvar ! So there is no need to add a external validator, right?

Reply:
Bingo.